components/klaudsolcms/pagination/contentPaginationVariables.js

```diff
@@ -0,0 +1,4 @@
+export const maximumNumberOfPage = 10; 
```

Comment:
Kindly move this file to constants folder and export it to constants folder > index.js